pom.xml

```diff
@@ -11,7 +11,7 @@
     </parent>
 
     <artifactId>gridcapa-starter-minio-adapter</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>0.0.3-SNAPSHOT</version>
```

Comment:
Why need you to come back to a previous version?

Reply:
The latest version is 0.0.2 actually, thats why